packages/pwa-buildpack/lib/Utilities/getServiceWorkerConfig.js

```diff
@@ -48,7 +48,8 @@ module.exports = function({
                 mode,
                 paths,
                 injectManifest: true,
-                enableServiceWorkerDebugging,
+                enableServiceWorkerDebugging: !!projectConfig.env
+                    .DEV_SERVER_SERVICE_WORKER_ENABLED,
```

Comment:
What does this do in production?

Also, can we use the new syntax here? Ex: `projectConfig.section('dev').serviceWorkerEnabled`

Reply:
The value of `DEV_SERVER_SERVICE_WORKER_ENABLED` is not valid in production. It is only used in development.

Reply:
Regarding the `.section` syntax, I was not able to find any resources around it. Any docs or links I can refer to?

Reply:
> Regarding the .section syntax, I was not able to find any resources around it. Any docs or links I can refer to?

I don't know if it's documented anywhere, I just saw this comment from @zetlen recently:
https://github.com/magento/pwa-studio/pull/1881#issuecomment-542305047

Reply:
Not sure why we have this API but I am not comfortable with it. If we need to change the config param's name or global search for it, things get hard. Also, what benefit is it adding here?

But that aside, the change is done. We will be using `projectConfig.section('dev').serverServiceWorkerEnabled` instead of `projectConfig.env.DEV_SERVER_SERVICE_WORKER_ENABLED`

Reply:
Here is the what and why of `.section` syntax: https://magento.github.io/pwa-studio/pwa-buildpack/configuration-management/